python/lsst/daf/butler/registry/databases/sqlite.py

```diff
@@ -398,6 +398,12 @@ def constant_rows(
         ]
         return sqlalchemy.sql.union_all(*selects).alias(name)
 
+    def get_constant_rows_max(self) -> int:
+        # Docstring inherited.
+        # This is the default SQLITE_MAX_COMPOUND_SELECT (see
+        # https://www.sqlite.org/limits.html):
+        return 500
```

Comment:
The default one (in `Database`) is 100, does it mean Postgres limit is lower than SQLite?

Reply:
There's no limit for this specifically listed on the PostgreSQL limits doc page, so I suspect the de facto PostgreSQL limit one on total SELECT statement size or number of bind parameters.  Keeping the limit really low here means we make temporary tables more often, which might be less efficient than we could be, but it guards pretty effectively against blowing one of those limits with really wide rows.

Or, in other words, "I don't think we know what we should set it to, but performance is okay with this."

Reply:
Oh, that's a good point, when you put it that way, and all of my testing with PostgreSQL was with the 1000-row limit before I made this change.  I think I will go change the default to that, as I think that's still plenty conservative enough.